pipeline_live/data/sources/alpaca.py

```diff
@@ -38,4 +41,5 @@ def fetch(symbols):
 
         return data
 
-    return parallelize(fetch, splitlen=199)(symbols)
+    with api:
```

Comment:
same here, fetch knows what `api` is, don't need the `with`

Reply:
Thanks I've updated the PR